tests/nlp/checks/train_test_validation/property_drift_test.py

```diff
@@ -0,0 +1,166 @@
+        # Arrange
+        train, _ = small_wikiann
+        train.calculate_default_properties()
+        check = PropertyDrift().add_condition_drift_score_less_than()
+        # Act
+        result = check.run(train_dataset=train, test_dataset=train)
+        condition_results = check.conditions_decision(result)
+        # Assert
+        assert len(condition_results) == 1
+        assert condition_results[0].is_pass() is True
+
+        assert_that(result.value, has_entries({
+            'Text Length': has_entries({'Drift score': 0.0, 'Method': 'Kolmogorov-Smirnov'}),
+            '% Special Characters': has_entries({'Drift score': 0.0, 'Method': 'Kolmogorov-Smirnov'}),
+            'Sentiment': has_entries({'Drift score': 0.0, 'Method': 'Kolmogorov-Smirnov'}),
+            'Average Word Length': has_entries({'Drift score': 0.0, 'Method': 'Kolmogorov-Smirnov'}),
+            'Subjectivity': has_entries({'Drift score': 0.0, 'Method': 'Kolmogorov-Smirnov'}),
+            'Max Word Length': has_entries({'Drift score': 0.0, 'Method': 'Kolmogorov-Smirnov'})
+        }))  # type: ignore
+
+
+    def test_with_drift(self, small_wikiann: t.Tuple[TextData, TextData]):
+        # Arrange
+        train, test = small_wikiann
+
+        train.calculate_default_properties(
+            include_long_calculation_properties=False
+        )
+        test.calculate_default_properties(
+            include_long_calculation_properties=False
+        )
+
+        check = PropertyDrift().add_condition_drift_score_less_than()
+
+        # Act
+        result = check.run(train_dataset=train, test_dataset=test)
+        condition_results = check.conditions_decision(result)
+
+        # Assert
+        assert len(condition_results) == 1
+        assert condition_results[0].is_pass() is False
+
+        assert_that(result.value, has_entries({
+            'Max Word Length': has_entries({'Drift score': 0.18000000000000005, 'Method': 'Kolmogorov-Smirnov'}),
+            'Average Word Length': has_entries({'Drift score': 0.1, 'Method': 'Kolmogorov-Smirnov'}),
+            '% Special Characters': has_entries({'Drift score': 0.16000000000000003, 'Method': 'Kolmogorov-Smirnov'}),
+            'Text Length': has_entries({'Drift score': 0.30000000000000004, 'Method': 'Kolmogorov-Smirnov'}),
+            'Subjectivity': has_entries({'Drift score': 0.14, 'Method': 'Kolmogorov-Smirnov'}),
+            'Sentiment': has_entries({'Drift score': 0.08000000000000007, 'Method': 'Kolmogorov-Smirnov'})
+        }))  # type: ignore
+
+
+class TestLabelClassification:
+
+    def generate_dataset(self):
+        return TextData(
+            raw_text=[
+                random.choice(['I think therefore I am', 'I am therefore I think', 'I am'])
+                for _ in range(20)
+            ],
+            label=[
+                random.choice([[0, 0, 1], [1, 1, 0], [0, 1, 0]])
+                for _ in range(20)
+            ],
+            task_type='text_classification'
+        )
```

Comment:
Put this one at context as 'dummy multilabel dataset'

Reply:
fixed